### ISSUE TITLE
Variable inspector not working

### ISSUE THREAD

**Issue by ktavabi** [ipython-contrib/jupyter_contrib_nbextensions#1133]
After enabling `varInspector/main` I am not seeing anything in the toolbar nor can I toggle the inspector via the command palette. I can see and change inspector parameters in the extensions configurator, and have set `True` to display window at startup, but there is just nothing visible, no errors or no sign of the extension in action.

**Comment by jcb91** [comment]
almost certainly a result of bug introduced in #1123, fixed in #1128. I've just pushed 0.3.3 to pypi which has this fixed, am in the process of updating the conda-forge build. What version are you using?

**Comment by ktavabi** [comment]
Version of nbextensions? Here is my entire Jupyter system in a python 2.7 conda virutal env:

> jupyter 1.0.0
> jupyter-contrib-core 0.3.3
> jupyter-contrib-nbextensions 0.3.2
> jupyter-highlight-selected-word 0.0.11
> jupyter-latex-envs 1.3.8.4             
> jupyter-nbextensions-configurator 0.2.8
> jupyter_client 5.1.0
> jupyter_console 5.1.0
> jupyter_core 4.3.0

**Comment by ktavabi** [comment]
Thanks! Seems like `pip install jupyter_contrib_nbextensions --upgrade` worked, although call ended with 

> OSError: [Errno 2] No such file or directory: '/home/.../miniconda3/envs/bad-baby/lib/python2.7/site-packages/setuptools-27.2.0-py2.7.egg'

**Comment by jcb91** [comment]
> Thanks! Seems like `pip install jupyter_contrib_nbextensions --upgrade worked`

Great :) Not sure about that `OSError`, I'm afraid, but if it all looks like it's working, then I expect it's fine.

Unfortunately I'm still waiting on Travis to [build the conda-forge package for 0.3.3](https://travis-ci.org/conda-forge/jupyter_contrib_nbextensions-feedstock/builds/290131409), but once it does, that should hopefully work too.

**Comment by jcb91** [comment]
I'll leave this issue open for now though, for the benefit of anyone installing 0.3.2 from conda :disappointed:

**Comment by SteliosStefanatos** [comment]
Same issue here. Just installed 0.3.2 from conda and many extensions do not appear in the toolbar even though enabled, including variable inspector, toc2, freeze, and runtools

**Comment by jcb91** [comment]
Yes, I'm sorry, but quite a few will be affected by this bug. For now (until the conda build completes correctly), the best I can offer is to try a pip upgrade. This is perfectly safe to do over the top of a conda install, but I forgot earlier that two steps are needed. To apply over the top of a conda install, use (with the relevant conda environment active)

    pip install --upgrade jupyter_contrib_nbextensions
    jupyter contrib nbextension install --sys-prefix

You may wish to add a --symlink flag to the second command (and you're not on Windows, which doesn't support symlinks).

**Comment by jcb91** [comment]
See also (same bug) #1127

**Comment by SteliosStefanatos** [comment]
@jcb91 Thanks a lot for the instructions, they work nicely! Really appreciate the team's efforts!

**Comment by jcb91** [comment]
@SteliosStefanatos you're welcome, glad to hear you got it fixed ok :)
I've just merged the [pr to update the conda repo](https://github.com/conda-forge/jupyter_contrib_nbextensions-feedstock/pull/23), so builds for 0.3.3 should hopefully get uploaded to the conda-forge channel soon once the various CI systems build them. You should be able to go back to using the conda package (as opposed to the pypi/pip one) using

    conda upgrade jupyter_contrib_nbextensions

But there isn't really any need to for now, since you've already got 0.3.3 anyway :)

**Comment by PPauls** [comment]
Hi, I have the same issue, even though all my versions are up to date... My jupyter is 5.6.0 and the nb extension's version is 0.5.0. Any ideas?

**Comment by jrjbertram** [comment]
Also seeing this with a fresh install of jupyter, nbextensions.  Can't seem to get the inspector window at all... trying it out for the first time, so might be missing something obvious.  (Enabled via command line originally, went back and verified through configurator that it was present and enabled.  selected the show window on startup...  disabled and reenabled the variable inspector via the configurator.  Stopped my notebook and jupyter lab...  restarted it, still no dice.)

**Comment by Fraixxer** [comment]
Hi all,

I just switch to jupyterlab notebook and would like to know how one can copy variables from the the variable inspector alike spyder variable explorer. Thanks.
